test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -176,4 +185,35 @@ private String executeAndRead(String command, Object... args)
             throw new RuntimeException(e);
         }
     }
+
+    public void preserveClusterLogs(String testName)
```

Comment:
I'm not opposed to extracting it, just curious about the reasoning. Though I do feel if it's going to be extracted like this, maybe the inner function shouldn't be public?